source/unified-test-format/unified-test-format.rst

```diff
@@ -2363,6 +2417,35 @@ An example of this operation follows::
         connections: 1
 
 
+Special Placeholder Value
+-------------------------
+
+$$placeholder
+~~~~~~~~~~~~~
+
+Syntax::
+
+  { field: { $$placeholder: {} } }
```

Comment:
I don't think the value is important, and would say as much below. I'd propose using `1` to simplicity, which is consistent with examples in the server manual (e.g. some commands disregard the value for a command name).

```suggestion
  { field: { $$placeholder: 1 } }
```

Reply:
Noted. I see this complicated the `stringOrPlaceholder` definition, which wasn't my intention -- but it's done now so I think we can leave it in place.

If any other context in the schema needs to use `$$placeholder` in the future, it will hopefully be extracted into a separate definition (out of `stringOrPlaceholder`).